pytest.ini

```diff
@@ -1,2 +1,3 @@
 [pytest]
 addopts = --ignore docs/xpd_examples.ipynb --ignore docs/xpd_examples2.ipynb --ignore docs/parallel_example.ipynb --ignore docs/dask_averaging.ipynb --ignore docs/inspection.ipynb --ignore docs/iterate_trains.ipynb
+filterwarnings = ignore::DeprecationWarning
```

Comment:
I think tests should make deprecation warnings visible in general. If we need to suppress specific ones, there are ways to temporarily filter warnings, filter warnings from specific modules, and even assert that [specific warnings are triggered](https://docs.pytest.org/en/latest/warnings.html#ensuring-code-triggers-a-deprecation-warning).

Reply:
fair enough